### ISSUE TITLE
Possible deadlock in the run_command_text

### ISSUE THREAD

**Issue by aleksey-novikov** [No9/libcrio#2]
In the Linux pipe buffer has restricted by 65 535 bytes by default. If the subprocess command has a lot of output (more than 64K), for example `crictl img -o json`, then its blocked until parent process read output. The [wait method](https://github.com/No9/libcrio/blob/5f2eacc939f0d74875c9132f70f851fcc22e0d4a/src/lib.rs#L327) is never returns.

**Comment by No9** [comment]
Thanks very much for this - Looking at this https://github.com/rust-lang/rust/issues/45572 there is cold comfort in the fact others have walked into this. 
There is a now a sample fix in https://doc.rust-lang.org/std/process/index.html - I'll review and put it in.

**Comment by aleksey-novikov** [comment]
Do you mean
`let output = child.wait_with_output().expect("Failed to wait ...");`
May be, I'm not good at this. This is the first project on Rust that I need to patch

**Comment by No9** [comment]
Yes - I'm writing a test for it and making the code change - Hold tight!!

**Comment by No9** [comment]
Fixed in https://github.com/No9/libcrio/releases/tag/v2.0.0
